src/xrGame/DiscordRPC.cpp

```diff
@@ -0,0 +1,35 @@
+#include "stdafx.h"
+#include "DiscordRPC.h"
+#include <ctime>
+
+DiscordRPC* g_DiscordRPC;
+
+void DiscordRPC::Init()
+{
+    DiscordEventHandlers Handle;
+    memset(&Handle, 0, sizeof(Handle));
+    Discord_Initialize("869824972464488510", &Handle, 1, NULL);
```

Comment:
1. Using NULL is outdated, it should be nullptr.
2. It's better to make a constant on top of the file (after includes) instead of hardcoded string with discord app ID.